src/js/list-remove-handlers.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-cycle */
+import { data as globalData, getFromLocalStorage, saveToLocalStorage } from './updateMethods.js';
```

Comment:
- [ ] As I mentioned in the previous comment,  kindly make sure you fix your linters errors instead of disabling them this is for best practices reasons and clean code. I suggest you apply this approach to all files of your app